submissions/f3d0t/task_js_dom/script.js

```diff
@@ -42,23 +42,20 @@ const createMenuList = (arrayOfPokemons, container) => {
 	container.appendChild(wrapper);
 };
 
-const getPokemonContent = (id) => {
+const createPokemonContent = (pokemon) => {
```

Comment:
Why you changed naming for function ?

Reply:
I was thinking about that this function actually creates new element that then appends to the container. But now i see that previous naming was more suitable. Should I change it back?

Reply:
there is no need for it now